plone/app/layout/views/configure.zcml

```diff
@@ -4,4 +4,13 @@
     xmlns:zcml="http://namespaces.zope.org/zcml"
     >
 
+  <browser:page
+      name="confirm-action"
+      for="*"
+      class=".views.ConfirmView"
```

Comment:
I would use the original class, given that what we want to have here is just a different template:
```suggestion
      class="plone.protect.views.ConfirmView"
```